addons/chemical/functions/fnc_addLoadAction.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: DiGii
+ * 
+ * Arguments:
+ * NONE
+ *
+ * Return Value:
+ * NONE
+ *
+ * Example:
+ * [] call kat_chemical_fnc_addLoadAction;
+ *
+ * Public: No
+ *
+ * This adds the function to Load gas mortar rounds into a mortar using the CSW function from ace
+*/
+private _condition = { //IGNORE_PRIVATE_WARNING ["_target", "_player"];
+	// If magazine handling is enabled or weapon assembly/disassembly is enabled we enable ammo handling
+	if ((ace_csw_ammoHandling == 0) && {!([false, true, true, ace_csw_defaultAssemblyMode] select (_target getVariable [ace_csw_assemblyMode, 3]))}) exitWith { false };
+	[_player, _target, ["isNotSwimming", "isNotSitting"]] call ace_common_canInteractWith
```

Comment:
missing ";"

Reply:
you are using _player, _target but where are they defined? missing params?

Reply:
it just knowed it